basics/getting_started.md

```diff
@@ -3,18 +3,22 @@ layout: contribute
 title: Getting Started
 ---
 
-# Setting up your coding environment
+# Getting started
 
-For now we have support for IntelliJ. We don't have IDE support for other
-languages in Bazel right now.
+This page helps you set up your coding environment, describes creating an
+IntelliJ project and takes you through compiling and debugging your project.
+Finally, you can read about Bazel's source code, how to navigate through it, and
+how to use the Bazel CI system.
 
-## Preparations
+## Installing Bazel
 
 *  [Install Bazel](https://bazel.build/versions/master/docs/install.html) on your
-   system. Note that for developing Bazel, you need the latest released version
-   of Bazel.
+   system. Note that you need the latest released version of Bazel for
+   development.
 *  Clone Bazel's Git repository from GitHub:
+
    *  `git clone https://github.com/bazelbuild/bazel.git`
```

Comment:
This shouldn't be an indented bullet, but just a code block as part of the parent list item. It's part of that task, not a sub-task. 

also, I think you can specify the type of code block, such as  "```bash"

```
git clone https://github.com/bazelbuild/bazel.git
```

Reply:
Removed bullet.